scripts/shell-test.sh

```diff
@@ -59,7 +59,10 @@ run_test inotify_cmd
 inotify / '' /bin/cat /dev/null >/dev/null && pass || fail
 
 run_test default_nic_mtu
-[[ -f "/sys/class/net/$(route -n | grep -E '^0\.0\.0\.0\s+\S+\s+0\.0\.0\.0' | grep -oE '\S+$')/mtu" ]] && pass || fail
+[[ -f "/sys/class/net/$(route -n | grep -E '^0\.0\.0\.0\s+\S+\s+0\.0\.0\.0' | sort -n -k5,5 | grep -oE '\S+$')/mtu" ]] && pass || fail
```

Comment:
Are we able to avoid duplicating the business logic in the test?

Reply:
Not easily. The purpose here is to make sure the file in /sys/class/net/x/mtu is "typically" available. I want to have the "x" extraction as close as possible to the actual logic.

Reply:
I should add a `head` here though, or in a system with multiple default routes this test fails.

Reply:
It turns out not possible to add a head, because we're restricted to whatever commands available in the image.